src/Mod/Draft/Resources/ui/preferences-dxf.ui

```diff
@@ -80,10 +83,10 @@ The C++ importer is faster, but has not as many features yet.</string>
         <item>
          <widget class="Gui::PrefCheckBox" name="checkBox_3">
           <property name="toolTip">
-           <string>Allow FreeCAD to download the Python converter for DXF import and export.
-You can also do that manually, by visiting
-https://github.com/yorikvanhavre/Draft-dxf-importer
-</string>
+           <string>Allow FreeCAD to download the Python converter for
+DXF import and export. You can also do this manually using the  
+Draft-DXF-importer. It can be aquired from:
+https://github.com/yorikvanhavre/Draft-dxf-importer</string>
```

Comment:
> Allow FreeCAD to download the Python converter for DXF import and export. You can also do this manually by installing the "dxf_library" workbench from the Addon Manager, or by getting the files from https://github.com/yorikvanhavre/Draft-dxf-importer

Reply:
Just remove the url or substitute with additional text ? If the latter, then what text do you propose?

Reply:
Substitute with additional text. I think it's important to mention that it can be installed from the Addon Manager. Adding the URL explicitly is maybe not necessary, as the both the DXF importer and the Addon Manager already connect to this address.

I already wrote my proposed text.

Reply:
Updated: 1df4abe